docs/cudf/source/groupby.rst

```diff
@@ -0,0 +1,234 @@
++------------------------------------+-----------+------------+----------+---------------+--------+----------+------------+-----------+
+| quantile                           | ✅        | ✅         |          |               |        |          |            |           |
++------------------------------------+-----------+------------+----------+---------------+--------+----------+------------+-----------+
+| median                             | ✅        | ✅         |          |               |        |          |            |           |
++------------------------------------+-----------+------------+----------+---------------+--------+----------+------------+-----------+
+| nunique                            | ✅        | ✅         | ✅       | ✅            |        |          |            | ✅        |
++------------------------------------+-----------+------------+----------+---------------+--------+----------+------------+-----------+
+| nth                                | ✅        | ✅         | ✅       |               |        |          |            | ✅        |
++------------------------------------+-----------+------------+----------+---------------+--------+----------+------------+-----------+
+| collect                            | ✅        | ✅         | ✅       |               | ✅     |          |            | ✅        |
++------------------------------------+-----------+------------+----------+---------------+--------+----------+------------+-----------+
+| unique                             | ✅        | ✅         | ✅       | ✅            |        |          |            |           |
++------------------------------------+-----------+------------+----------+---------------+--------+----------+------------+-----------+
+
+GroupBy apply
+-------------
+
+To apply function on each group, use the ``GroupBy.apply()`` method:
+
+.. code:: python
+
+    >>> df
+       a  b  c
+    0  1  1  1
+    1  1  1  2
+    2  1  2  3
+    3  2  2  4
+    4  2  3  5
+    >>> df.groupby('a').apply(lambda x: x.max() - x.min())
+       a  b  c
+    a
+    0  0  1  2
+    1  0  1  1
+
+Limitations
+~~~~~~~~~~~
+
+-  ``apply`` works by applying the provided function to each group
+   sequentially, and concatenating the results together. **This can be
+   very slow**, especially for a large number of small groups. For a
+   small number of large groups, it can give acceptable performance
+
+-  The results may not always match Pandas exactly. For example, cuDF
+   may return a ``DataFrame`` containing a single column where Pandas
+   returns a ``Series``. Some post-processing may be required to match
+   Pandas behavior.
+
+-  cuDF does not support some of the exceptional cases that Pandas
+   supports with ``apply``, such as
+   ```describe`` <https://pandas.pydata.org/pandas-docs/stable/user_guide/groupby.html#flexible-apply>`__.
```

Comment:
This doesn't render properly in sphinx. Tried a bit but sill not sure how to create a link for inline code. We could fall back to just regular links:
```suggestion
   `describe <https://pandas.pydata.org/pandas-docs/stable/user_guide/groupby.html#flexible-apply>`_.
```

Reply:
Good catch, I didn't notice this. I found a way to get it working.